### PR TITLE
Update gltf2material graph pbr node

### PR DESCRIPTION
**Pull request by jjcasmar**
The .qt3d file was using an old prototype and when exporting to a .frag.json, it failed. This patch updates the .qt3d file and the .frag.json created with the new .qt3d file.

### CODE REVIEW

File: src/core/shaders/graphs/metallicroughness.frag.json [hunk 1]
```diff
@@ -1924,17 +1926,6 @@
                     ],
                     "substitution": "vec4 $outputColor = kuesa_metalRoughFunction($baseColor, $metalness, $roughness, $ambientOcclusion, $worldPosition, $worldView, $worldNormal);"
                 },
-                {
```
Comment (mkrus) [inline src/core/shaders/graphs/metallicroughness.frag.json]: why are we loosing these? would that not break ES2 PBR support?

File: src/core/shaders/graphs/metallicroughness.frag.json [hunk 2]
```diff
@@ -1924,17 +1926,6 @@
...
-                    "format": {
```
Comment (mkrus) [inline src/core/shaders/graphs/metallicroughness.frag.json]: ditto

File: src/core/shaders/graphs/metallicroughness.frag.json [hunk 3]
```diff
@@ -2140,28 +2133,6 @@
                 "matrix"
             ],
             "rules": [
-                {
```
Comment (mkrus) [inline src/core/shaders/graphs/metallicroughness.frag.json]: ditto

File: src/core/shaders/graphs/metallicroughness.frag.json (hunk 2 above)
Comment (jjcasmar) [reply src/core/shaders/graphs/metallicroughness.frag.json]: I have load the prototypes in the kuesa folder, open the .qt3d file and save/export. Maybe the prototypes in the repo are incorrect?

File: src/core/shaders/graphs/metallicroughness.frag.json [hunk 4]
```diff
@@ -1705,7 +1705,7 @@
                         "major": 2,
                         "minor": 0
                     },
-                    "substitution": "highp $type $output; if ($condition) discard; $output = $value;"
+                    "substitution": "$type $output; if ($condition) discard; $output = $value;"
```
[the remark this answers is not in the document]
Comment (jcelerier) [reply src/core/shaders/graphs/metallicroughness.frag.json]: just checked and this is a bug in the nodeeditor, just pushed a change for it

[the remark this answers is not in the document]
Comment (jcelerier) [reply src/core/shaders/graphs/metallicroughness.frag.json]: (interestingly, `if` also does not have it there for GLES2 too, I guess that's a bug)

**Comment by jcelerier** [comment]
(see also https://codereview.kdab.com/#/c/65292/)

File: src/core/shaders/graphs/metallicroughness.frag.json (hunk 1 above)
Comment (jcelerier) [reply src/core/shaders/graphs/metallicroughness.frag.json]: done

File: src/core/shaders/graphs/metallicroughness.frag.json (hunk 2 above)
Comment (jcelerier) [reply src/core/shaders/graphs/metallicroughness.frag.json]: done (by updating the prototypes both in the repo and the nodeeditor)

File: src/core/shaders/graphs/metallicroughness.frag.json (hunk 3 above)
Comment (jcelerier) [reply src/core/shaders/graphs/metallicroughness.frag.json]: done

**Pull request merged**